integration-tests/src/test/java/org/zowe/apiml/functional/apicatalog/ApiCatalogAuthenticationTest.java

```diff
@@ -115,6 +116,20 @@ void givenValidBasicAuthentication(String endpoint, Request request) {
                     .statusCode(is(SC_OK));
             }
 
+            @ParameterizedTest(name = "givenValidBearerAuthentication {index} {0} ")
+            @MethodSource("org.zowe.apiml.functional.apicatalog.ApiCatalogAuthenticationTest#requestsToTest")
```

Comment:
Should these tests be done for gateway and discovery too?

Reply:
Done!